libnspr.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: libnspr
+  version: 4.35
+  epoch: 0
+  description: "Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level and libc-like functions."
+  copyright:
+    - license: Mozilla Public License v2
+  target-architecture:
+    - x86_64
+    - aarch64
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - mercurial
+      - bash
+
+pipeline:
+  # we need to use runs because melange dont have a mercurial checkout pipeline yet
+  - runs: |
+      hg clone https://hg.mozilla.org/projects/nspr
```

Comment:
this is cool! I think you'll need to template in some logic though to ensure we checkout the right version from up above, and potentially pin the commit (not entirely sure how mercurial does this), otherwise we'll get a different version everytime this runs.

Reply:
I found in this repo we have the [follow tags](https://hg.mozilla.org/projects/nspr/tags), with `-r` flag we can checkout to specific tag.

Reply:
nice!